.github/workflows/clean_workflow.yml

```
@@ -20,12 +20,19 @@ on:
         required: true
       DEV_KUBE_CONFIG:
         required: true
+      bingo_token:
+        required: true
```

I think BINGO_REPO_TOKEN is better to use for it
```suggestion
      BINGO_REPO_TOKEN:
        required: true
```